### PR TITLE
Default every false instance of ExcludeUserAudio to true

### PR DESCRIPTION
**Pull request by ninruB**
## About the PR

Fixes: #31433 (?)

Please ELI5 in reviews if possible.

## Technical details

I have no idea if I'm doing this right, client/server compiled on debug and no errors were seen.

## Requirements

- [X] I have read and I am following the [Pull Request Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html). I understand that not doing so may get my pr closed at maintainer’s discretion
- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

N/A (?)

### CODE REVIEW

**Comment by Hreno** [comment]
Wherever true is explicitly passed, you can remove it. And wherever no argument is passed in, you now need to pass false explicitly.

**Comment by ninruB** [comment]
> Wherever true is explicitly passed, you can remove it. And wherever no argument is passed in, you now need to pass false explicitly.

I'll figure it out tomorrow.

File: Content.Shared/Containers/ItemSlot/ItemSlotsSystem.cs [hunk 1]
```diff
@@ -227,7 +227,7 @@ private void OnInteractUsing(EntityUid uid, ItemSlotsComponent itemSlots, Intera
                 if (slot.Item != null)
                     _handsSystem.TryPickupAnyHand(args.User, slot.Item.Value, handsComp: hands);
 
-                Insert(uid, slot, args.Used, args.User, excludeUserAudio: true);
+                Insert(uid, slot, args.Used, args.User, excludeUserAudio: default);
```
Comment (Hreno) [inline Content.Shared/Containers/ItemSlot/ItemSlotsSystem.cs:230]: `default` will become false here, it's a breaking change. You should just delete the entire `excludeUserAudio` argument here.

Also in other places.

File: Content.Shared/Containers/ItemSlot/ItemSlotsSystem.cs [hunk 2]
```diff
@@ -300,27 +300,27 @@ public bool CanInsert(EntityUid uid, EntityUid usedUid, EntityUid? user, ItemSlo
...
+        public bool TryInsert(EntityUid uid, string id, EntityUid item, EntityUid? user, ItemSlotsComponent? itemSlots = null, bool excludeUserAudio = true)
         {
             if (!Resolve(uid, ref itemSlots))
                 return false;
 
             if (!itemSlots.Slots.TryGetValue(id, out var slot))
                 return false;
 
-            return TryInsert(uid, slot, item, user, excludeUserAudio: excludeUserAudio);
+            return TryInsert(uid, slot, item, user, excludeUserAudio: default);
```
Comment (Hreno) [inline Content.Shared/Containers/ItemSlot/ItemSlotsSystem.cs:311]: You must forward the `excludeUserAudio` as was passed in to this method, cannot override it to `default` (which is false, for boolean).

Also in other places.

**Review by Hreno — CHANGES_REQUESTED** [review]

**Comment by ninruB** [comment]
Thanks, I addressed the review, should be good now. Going to re-request a review just to be sure.

File: Content.Shared/Containers/ItemSlot/ItemSlotsSystem.cs [hunk 3]
```diff
@@ -473,7 +473,7 @@ public bool TryEject(EntityUid uid, ItemSlot slot, EntityUid? user, [NotNullWhen
             if (user != null && item != null && !_actionBlockerSystem.CanPickup(user.Value, item.Value))
                 return false;
 
-            Eject(uid, slot, item!.Value, user, excludeUserAudio);
+            Eject(uid, slot, item!.Value, user, excludeUserAudio = false);
```
Comment (Hreno) [inline Content.Shared/Containers/ItemSlot/ItemSlotsSystem.cs:476]: This shouldn't be done, keep `excludeUserAudio` as is.

**Review by Hreno — CHANGES_REQUESTED** [review]
Have you searched for usages of every of these methods across the codebase and made sure the wrong value isn't now passed in any more? Remember, functionality during a refactoring must not change.

**Comment by Hreno** [comment]
All usages weren't fixed.

Can I suggest you to tackle an easier issue where you'd have a higher confidence in making things work correctly? Not that I have 100% confidence in what I contribute is correct, but I'm reasonably assured.

Changing things blindly and hoping they will be fine is a recipe for failure.

**Comment by ninruB** [comment]
> Can I suggest you to tackle an easier issue where you'd have a higher confidence in making things work correctly?

The previous issue I tried to tackle marked as beginner friendly and easy ended up actually being a landmine, so I'm not too eager to switch. I'll give this one a way more rigorous look, I'll re-request a review when I'm absolutely certain this changes no functionality.

**Comment by ninruB** [comment]
I've had my schedule extended by a LOT, closing PR as per lack of free time and or dedication.
